src/util/log.cpp

```diff
@@ -75,21 +75,29 @@ SimpleLogger::~SimpleLogger()
         switch (level)
         {
         case logWARNING:
-            std::cerr << (is_terminal ? RED : "") << os.str() << (is_terminal ? COL_RESET : "")
-                      << std::endl;
+        case logERROR:
+            std::cerr << buffer.str();
```

Comment:
@MoKob ah, good point - there was some implied behaviour here.  The base class assumes std::cout/cerr are the final destination, but subclasses can override that.

I've clarified the logic.  `Log` still finishes up with cerr/cout, but subclasses that override the stream will use that instead.